api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1161,6 +1202,9 @@ public Boolean hasPreviousVersion() {
 	 * @see Auditable#setCreator(User)
 	 */
 	@Override
+	@Access(AccessType.PROPERTY)
```

Comment:
This is already mapped in the superclass

Reply:
True.I have noticed.